Replaced `o_` prefixes and modified method names

aashan10 opened the pull request:
Resolves #54

aashan10 committed:
Replace o_ prefixes in Model

aashan10 committed:
Replaced o_ prefixes

aashan10 committed:
Replaced non existant method with correct name

dvesh3 commented on src/Tools/Installer.php:
```diff
@@ -32,11 +32,11 @@ public function install()
               `id` int(11) NOT NULL AUTO_INCREMENT,
               `o_id` int(11) NOT NULL,
-              `o_classId` varchar(50) NOT NULL,
+              `classId` varchar(50) NOT NULL,
```
we also need a migration for updating existing table

aashan10 replied:
Added the migrations for this one

dvesh3 commented on src/Tools/Installer.php:
```diff
@@ -32,11 +32,11 @@ public function install()
         $db->executeQuery('CREATE TABLE `' . Dao::TABLE_NAME . '` (
               `id` int(11) NOT NULL AUTO_INCREMENT,
               `o_id` int(11) NOT NULL,
```
`o_id` is not replaced?

aashan10 replied:
Well this one refers to the object id on `objects` table and since this table already has an `id` column, it does not make sense to remove the `o_` prefix.

aashan10 committed:
Added migration for migrating the removal of o_ prefixed columns

aashan10 committed:
Apply php-cs-fixer changes

aashan10 committed:
Fixed mismatching types

aashan10 committed:
Fixed Typo

Merge branch 'issue-54' of https://github.com/aashan10/output-data-config-toolkit into issue-54

aashan10 committed:
Apply php-cs-fixer changes

dvesh3 commented on src/Controller/ClassController.php:
```diff
@@ -127,7 +127,7 @@ private function considerClassificationStoreForColumnConfig(Request $request, ?D
                 try {
                     // @todo: is there a better way to check if a classification group is assigned to the class?
-                    $enrichment = Db::get()->fetchOne("SELECT EXISTS (SELECT * FROM object_classificationstore_groups_{$class->getId()} WHERE o_id = '{$targetObjectId}')");
+                    $enrichment = Db::get()->fetchOne("SELECT EXISTS (SELECT * FROM object_classificationstore_groups_{$class->getId()} WHERE id = '{$targetObjectId}')");
```
why not using `Service::getVersionDependentDatabaseColumnName`?

dvesh3 commented on src/OutputDefinition/Listing/Dao.php:
```diff
@@ -37,13 +37,13 @@ public function load(): array
 
         $unitIds = $this->db->fetchAllAssociative(
-            'SELECT o_id, id, o_classId, channel FROM ' . OutputDefinition\Dao::TABLE_NAME .
+            'SELECT o_id, id, classId, channel FROM ' . OutputDefinition\Dao::TABLE_NAME .
```
`Service::getVersionDependentDatabaseColumnName`

dvesh3 replied:
Sorry, this should stay like this since bundle table column is upgraded in migration

dvesh3 reviewed (changes requested):
@aashan10 please see requested changes

aashan10 committed:
Replaced o_ prefixes with version aware column names

aashan10 committed:
Bumpped pimcore/pimcore dependency to 10.6

aashan10 committed:
Apply php-cs-fixer changes

dvesh3 commented on composer.json:
```diff
@@ -2,8 +2,10 @@
+  "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^10.5 || ^11@dev",
+    "pimcore/pimcore": "^10.6 || ^11@dev",
```
```suggestion
    "pimcore/pimcore": "^10.6 || ^11.0",
```
since minimum stability now set to dev

dvesh3 committed:
Update composer.json

kingjia90 commented on src/Controller/AdminController.php:
```diff
@@ -105,8 +105,8 @@ public function getOutputConfigsAction(Request $request)
                     'channel' => $translator->trans($channel, [], 'admin'),
-                    'object_id' => $def->getO_Id(),
-                    'is_inherited' => $def->getO_Id() != $objectId
+                    'object_id' => $def->getId(),
```
there's something that doesn't look right here, `object_id` would be the same as `id` on [line 105](https://github.com/pimcore/output-data-config-toolkit/pull/70/files#diff-15f7f071121bfa3ffa4bf6f5b31db091f12a2ce65acae1642ab42c45e0c9eef7R105)
The right method is [getObjectId()](https://github.com/pimcore/output-data-config-toolkit/pull/70/files#diff-4026997d615e34ef8e443eecd0b6cf4c96eaf729587e08872658afe9a26ee96dR148)

kingjia90 replied:
Proposing the fixes in https://github.com/pimcore/output-data-config-toolkit/pull/76
